packages/libs/error-stack/Cargo.toml

```diff
@@ -29,11 +30,13 @@ futures = { version = "0.3.21", default-features = false, features = ["executor"
 rustc_version = "0.2.3"
 
 [features]
-default = ["std"]
-std = ["anyhow?/std"]
+default = ["std", "small", "fancy"]
+std = ["anyhow?/std", "once_cell?/std"]
+fancy = []
```

Comment:
Thinking about this a little bit more: Probably we should just keep it as the logic is already implemented. I don't really have an opinion. Ideas?

Reply:
I thought what might be neat is the inclusion of display as a module and then provide several optional hooks people can use, like json, fancy, ascii, with ascii being the default.

That way people wouldn't repeat creating or copying the same code/example.

Reply:
@TimDiekmann what do you think?
I'd move display.rs to `hook/fancy.rs`, move `hook.rs` to `hook/mod.rs`, and then include a json one for `hook/json.rs`. This way I could also include `termcolor` without worrying too much about a new dependency and `fancy` would still be optional.

Reply:
I think this would be an option. I wonder if you want to stash your changes for `fancy` (and maybe `json` if you already did some work on it) and want to open another PR for that? Then the PRs would be a little bit more self-contained.